dev-python/logfury/logfury-0.1.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Boilerplate library for logging method calls"
+HOMEPAGE="https://github.com/ppolewicz/logfury https://pypi.org/project/logfury/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
```

Comment:
Nope, it's `BSD`.

project github says it's BSD 3-clause, 
https://github.com/ppolewicz/logfury/blob/master/LICENSE.txt
in gentoo it's called just `BSD`.

If you look into that file in licenses/BSD, it contains exactly 3 clauses =)